### PR TITLE
(! URGENT !) updated to read the classifier information from yaml file

### PR DESCRIPTION
**Pull request by pkorivi**
I made small changes to code such that we can read the classifier we are using information from yaml file along with the rest of the lap parameters.

### CODE REVIEW

File: ros/src/tl_detector/light_classification/tl_classifier.py [hunk 1]
```diff
@@ -3,9 +3,13 @@
 import numpy as np
 
 class TLClassifier(object):
-    def __init__(self):
-        #TODO load classifier
-        pass
+    def __init__(self, is_site, classifier):
+        self.is_site = is_site
+        self.classifier = classifier
+        print "is_site ",self.is_site
```
Comment (thejerrytan) [inline ros/src/tl_detector/light_classification/tl_classifier.py]: think we should standardize and all use rospy.loginfo / logdebug etc.

File: ros/src/tl_detector/light_classification/tl_classifier.py [hunk 2]
```diff
@@ -51,11 +55,19 @@ def get_classification(self, image, method):
             int: ID of traffic light color (specified in styx_msgs/TrafficLight)
 
         """
-        #TODO implement light color prediction
-        if(method == "opencv"):
+        #call classifier for the site - real data
+        if(self.is_site == True):
```
Comment (thejerrytan) [inline ros/src/tl_detector/light_classification/tl_classifier.py]: just 
```
if self.is_site:
```
will do rite?

**Comment by thejerrytan** [comment]
Apart from minor changes, LGTM, good job using yaml config file to switch between configuration options.

File: ros/src/tl_detector/light_classification/tl_classifier.py [hunk 3]
```diff
@@ -51,11 +55,19 @@ def get_classification(self, image, method):
...
+            if(self.classifier == "opencv"):
+                return self.simple_opencv_red_color_classifier(image)
+            elif(self.classifier == "dl_ssd"):
+                self.dl_ssd_classifier_carla(image)
+
+        #is_site is flase - call simulation classiier
```
Comment (ser94mor) [inline ros/src/tl_detector/light_classification/tl_classifier.py:70]: flase -> false

File: ros/src/tl_detector/site_traffic_light_config.yaml [hunk 4]
```diff
@@ -6,3 +6,4 @@ camera_info:
 stop_line_positions:
     - [8.0, 16.2]
 is_site: True
+classifier: "dl_ssd" # opencv, dl_ssd, simple_cnn, yolo
```
Comment (ser94mor) [inline ros/src/tl_detector/site_traffic_light_config.yaml]: Please, add a new line at the end of this file (GitHub suggests).

**Review by ser94mor — APPROVED** [review]

File: ros/src/tl_detector/light_classification/tl_classifier.py (hunk 2 above)
Comment (Karthikeya108) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: I second that

**Review by thejerrytan — APPROVED** [review]

File: ros/src/tl_detector/light_classification/tl_classifier.py (hunk 1 above)
Comment (pkorivi) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: Resolved

File: ros/src/tl_detector/light_classification/tl_classifier.py (hunk 2 above)
Comment (pkorivi) [reply ros/src/tl_detector/light_classification/tl_classifier.py]: Updated it

File: ros/src/tl_detector/site_traffic_light_config.yaml (hunk 4 above)
Comment (pkorivi) [reply ros/src/tl_detector/site_traffic_light_config.yaml]: updated

**Pull request merged**